README.md

```diff
@@ -0,0 +1,238 @@
+# GitHub Actions: sharing your secrets with third-party actions
+
+<!-- markdownlint-disable-next-line MD013 -->
+[![Default Organization Configuration](https://github.com/Nautilus-Cyberneering/github-actions-secrets/actions/workflows/default-config.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/github-actions-secrets/actions/workflows/default-config.yml) [![Disable GITHUB_TOKEN](https://github.com/Nautilus-Cyberneering/github-actions-secrets/actions/workflows/disable-github-token.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/github-actions-secrets/actions/workflows/disable-github-token.yml) [![Environment secret](https://github.com/Nautilus-Cyberneering/github-actions-secrets/actions/workflows/environment-secret.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/github-actions-secrets/actions/workflows/environment-secret.yml) [![MegaLinter](https://github.com/Nautilus-Cyberneering/github-actions-secrets/actions/workflows/mega-linter.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/github-actions-secrets/actions/workflows/mega-linter.yml) [![Print GitHub Context](https://github.com/Nautilus-Cyberneering/github-actions-secrets/actions/workflows/print-github-context.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/github-actions-secrets/actions/workflows/print-github-context.yml) [![Print env vars in docker](https://github.com/Nautilus-Cyberneering/github-actions-secrets/actions/workflows/test-env-action.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/github-actions-secrets/actions/workflows/test-env-action.yml)
+
+- [Setup the workflow with the minimum permissions](#setup-the-workflow-with-the-minimum-permissions)
+  - [Solution 1: Using CODEOWNERS configuration](#solution-1-using-codeowners-configuration)
+  - [Solution 2: Using branch protection rules and environment secrets](#solution-2-using-branch-protection-rules-and-environment-secrets)
+  - [Solution 3: Use a different repository to run MegaLinter](#solution-3-use-a-different-repository-to-run-megalinter)
+  - [Conclusion](#conclusion)
+- [How GitHub Actions can get access to secrets](#how-github-actions-can-get-access-to-secrets)
+- [Other questions](#other-questions)
+- [Links](#links)
+- [Hot to contribute](#how-to-contribute)
+
+We are using [MegaLinter](https://github.com/megalinter/megalinter) in most of our projects. It is a very useful GitHub Action to check your code, not only for basic things like the code format, but it also has some packages to detect security problems. But we have been wondering if the MegaLinter itself could be a security problem. This is the default workflow configuration when you install it:
```

Comment:
I would start an additiona paragraph like so:

...security problems. However, we have also been wondering if the MegaLinter itself could be a security problem. 

Now, start by looking at it's default workflow configuration when you install it: